openstack/keppel/values.yaml

```diff
@@ -179,7 +179,8 @@ pgmetrics:
             usage: "GAUGE"
             description: "UNIX timestamp of next scheduled manifest sync, or 0 if no replica repos have ever been synced"
     keppel_manifests:
-      query: "SELECT COUNT(*) AS count, EXTRACT(epoch FROM MIN(validated_at)) AS min_validated_at, SUM(CASE WHEN validation_error_message = '' THEN 0 ELSE 1 END) AS validation_errors, COALESCE(EXTRACT(epoch FROM MIN(next_vuln_check_at)), 0) AS min_next_vuln_check_at, SUM(CASE WHEN next_vuln_check_at > NOW() THEN 0 ELSE 1 END) AS pending_vuln_checks FROM manifests"
+      query: >
+        SELECT COUNT(*) AS count, EXTRACT(epoch FROM MIN(validated_at)) AS min_validated_at, SUM(CASE WHEN validation_error_message = '' THEN 0 ELSE 1 END) AS validation_errors, COALESCE(EXTRACT(epoch FROM MIN(vuln_info.next_check_at)), 0) AS min_next_vuln_check_at, SUM(CASE WHEN vuln_info.next_check_at > NOW() THEN 0 ELSE 1 END) AS pending_vuln_checks FROM manifests, vuln_info
```

Comment:
This should use an inner join instead of a cross join, otherwise `COUNT(*)` and `SUM()` will yield unexpected results.